src/components/popup/home/Tabs.tsx

```diff
@@ -79,6 +79,9 @@ const StyledTab = styled.div<{ active?: boolean; tabId: number }>`
   color: ${(props) =>
     props.active ? props.theme.primaryText : props.theme.secondaryTextv2};
```

Comment:
```suggestion
    props.active ? props.theme.primaryTextv2 : props.theme.secondaryTextv2};
```
`primaryText` is not rendering properly for me, and it would be better to use the new v2 colors anyway